### ISSUE TITLE
Excessive battery usage even without sessions. 

### ISSUE THREAD

**Issue by voronind-com** [nix-community/nix-on-droid#411]
Hi, I've been using nix on droid for a while. And since the last update my phone overheats and drains battery right after I launch nix app. It still drains battery even after I exit all the sessions (no notification) and is fixed either by reboot or force closing the app. 

![Screenshot_20240905-134250](https://github.com/user-attachments/assets/4fa792b1-eb59-49ca-816e-b80b599b8451)
![Screenshot_20240905-134305](https://github.com/user-attachments/assets/4e81c8c3-7a67-48fd-9e3c-db9adfc06c0a)

**Comment by t184256** [comment]
Not sure how to debug this. Does it happen with a clean install? Is the devices rooted? Are there any processes running with nix-on-droid uid when there are no sessions?

**Comment by voronind-com** [comment]
> Not sure how to debug this. Does it happen with a clean install? Is the devices rooted? Are there any processes running with nix-on-droid uid when there are no sessions?

I haven't tested a clean install. Device is not rooted, stock rom. Not sure about processes, but there shouldn't be any.

Should I test with a clean install now?

**Comment by t184256** [comment]
Yes, please, backup the data and test with a clean install and default config.
